models/AOP_10.ttl

```diff
@@ -7,7 +7,8 @@
     @base <http://www.w3.org/2002/07/owl#> .
 
     <https://noctua.apps.renci.org/model/AOP_10> a owl:Ontology .
-<https://noctua.apps.renci.org/model/AOP_10> <http://purl.org/dc/elements/1.1/title> "Parkinsonian motor deficits"^^xsd:string .
+<https://noctua.apps.renci.org/model/AOP_10> <http://purl.org/dc/elements/1.1/title> "209    Binding to the picrotoxin site of ionotropic G...
+Name: Title, dtype: object"^^xsd:string .
```

Comment:
@meisha I think there is a bug in your title output; see here and other files.

Reply:
@balhoff What is the bug you're seeing? The issues was that the titles were being assigned to the incorrect AOPs so they probably will be completely different.

Reply:
Do you mean for this text to be part of the title? `Name: Title, dtype: object`

Reply:
Aha! I didn't not mean for that to be included. I'll let you know when I fix that and the file name issue.